lib/api/auth.ts

```
@@ -0,0 +1,69 @@
+
+/**
+ * Example function to check if a user is an admin
+ * Replace this with your actual implementation
+ */
+async function checkIfUserIsAdmin(userId: string): Promise<boolean> {
+  // Example implementation - replace with your own
+  // e.g., fetch user from database and check roles
+  // For now, return false as placeholder
+  const adminUserIds = process.env.ADMIN_USER_IDS?.split(",") || [];
+  return adminUserIds.includes(userId);
+  return false;
```

Biome (1.9.4)

[error] This code is unreachable

... because this statement will return from the function beforehand


(lint/correctness/noUnreachable)